test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... ok
test_to_degrees (__main__.TestExercice) ... FAIL
test_to_farenheit (__main__.TestExercice) ... ok
test_to_radians (__main__.TestExercice) ... ok

======================================================================
FAIL: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 83, in test_to_degrees
    self.assertListEqual(
AssertionError: Lists differ: [(114.59155902616465, 35.493541569878744, 2[196 chars]998)] != [(79.0980174562859, 5.881047377154118, 29.6[197 chars]998)]

First differing element 0:
(114.59155902616465, 35.493541569878744, 29.612494192724625)
(79.0980174562859, 5.881047377154118, 29.612494192724625)

- [(114.59155902616465, 35.493541569878744, 29.612494192724625),
-  (-229.1831180523293, 49.01291686024251, 0.7750116145507491),
-  (286.4788975654116, 28.73385392469686, 44.031235481811564),
-  (234.3397382085067, 20.384292510402133, 23.057550624127998)]
+ [(79.0980174562859, 5.881047377154118, 29.612494192724625),
+  (-278.1960349125718, 48.23790524569176, 0.7750116145507491),
+  (257.74504364071475, -15.297381557114704, 44.031235481811564),
+  (213.95544569810457, -2.673258113725865, 23.057550624127998)] : Mauvais calcul de radians -> degres

----------------------------------------------------------------------
Ran 7 tests in 0.005s

FAILED (failures=1)
